helm/security-bundle/values.yaml

```diff
@@ -23,15 +23,15 @@ apps:
     chartName: falco
     catalog: giantswarm
     enabled: true
-    namespace: security-pack
+    namespace: security-bundle
```

Comment:
We should also loudly include this namespace change in customer communications. I think this will break some existing customer adaptations